cricket-batter-ranking/site/index.html

```
@@ -0,0 +1,138 @@
+  <button class="tab" data-view="explore">Explore (scatter)</button>
+  <button class="tab" data-view="entry">Entry context</button>
+  <button class="tab" data-view="about">About</button>
+</nav>
+
+<main>
+  <!-- RANKINGS -->
+  <section id="view-rankings" class="view active">
+    <div class="view-head">
+      <h2>The ranking</h2>
+      <p class="muted">Sort by any factor. Click a row to expand its full factor breakdown, or open the player's profile. Scores are 0–100 within the selected cohort. The composite is a <b>weighted blend of the eleven factor scores</b> — and the weights are yours to set below.</p>
+      <input id="search" type="search" placeholder="Filter players…" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)